lms/views/api/d2l/authorize.py

```diff
@@ -8,7 +8,7 @@
 from lms.validation.authentication import OAuthCallbackSchema
 
 GROUPS_SCOPES = ("groups:*:*",)
-FILES_SCOPES = ("content:toc:read",)
+FILES_SCOPES = ("content:toc:read", "content:topics:read")
```

Comment:
I wonder what will happen to people who've already granted us access tokens without the topics scope? Will they get prompted to re-authorize us? Maybe no real users are using this feature yet anyway?